browser-test/src/support/admin_questions.ts

```diff
@@ -308,28 +322,46 @@ export class AdminQuestions {
     await this.expectDraftQuestionExist(questionName, questionText);
   }
 
-  async addDropdownQuestion(questionName: string,
+  /** Fills out the form for a dropdown question and clicks submit.  */
+  async createDropdownQuestion(questionName: string,
     options: Array<string>,
     description = 'dropdown description',
     questionText = 'dropdown question text',
     helpText = 'dropdown question help text',
-    enumeratorName = AdminQuestions.DOES_NOT_REPEAT_OPTION) {
-    await this.gotoAdminQuestionsPage();
+    enumeratorName = AdminQuestions.DOES_NOT_REPEAT_OPTION){
+      await this.gotoAdminQuestionsPage();
 
-    await this.page.click('#create-question-button');
-    await this.page.click('#create-dropdown-question');
-    await waitForPageJsLoad(this.page);
-
-    await this.fillInQuestionBasics(questionName, description, questionText, helpText, enumeratorName);
+      await this.page.click('#create-question-button');
+      await this.page.click('#create-dropdown-question');
 
-    for (var index in options) {
-      await this.page.click('#add-new-option');
-      var matchIndex = Number(index) + 1;
-      await this.page.fill(`:nth-match(#question-settings div.flex-row, ${matchIndex}) input`, options[index]);
-    }
+      waitForPageJsLoad(this.page);
+  
+      await this.fillInQuestionBasics(questionName, description, questionText, helpText, enumeratorName);
+  
+      for (var index in options) {
+        await this.page.click('#add-new-option');
+        var matchIndex = Number(index) + 1;
+        await this.page.fill(`:nth-match(#question-settings div.flex-row, ${matchIndex}) input`, options[index]);
+      }
+      
+      await this.clickSubmitButtonAndNavigate('Create');
+  }
 
-    await this.clickSubmitButtonAndNavigate('Create');
+  /** Changes the input field of a multi option answer. */
+  async changeMultiOptionAnswer(index: number, text: string) {
+    await this.page.fill(`:nth-match(#question-settings div.flex-row, ${index}) input`, text);
+  }
 
+  /** Fills out the form for a dropdown question, clicks submit, and verifies the new question exists.  */
+  async addDropdownQuestion(questionName: string,

```

Comment:
suggestion (non-blocking): same suggestion as with `createRadioButtonQuestion`

Reply:
Fixed.

Reply:
It's still using positional args

Reply:
Oops, updated "createDropdownQuestion" instead. This function already existed and there are a lot similar to it, so I refactored the other functions as well, but put it in a separate PR because of how many of these function types there are - https://github.com/seattle-uat/civiform/pull/1632.  We can merge that into this branch or merge it into main after these changes go in.